packages/react-components/react-utilities/src/utils/isHTMLElement.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * verifies if a given node is an HTMLElement,
+ * this method works seamlessly with frames and elements from different documents
```

Comment:
Please add this link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms